src/Generator/Resolver/Value/Chainable/EvaluatedValueResolver.php

```diff
@@ -58,6 +58,10 @@ public function resolve(
         $_scope = $scope;
         try {
             $_scope['current'] = $fixture->getValueForCurrent();
+
+            if ($_scope['current'] instanceof FixtureInterface) {
+                $_scope['current'] = $fixtureSet->getObjects()->get($_scope['current'])->getInstance();
```

Comment:
what about setting the right value directly L60?

Reply:
With using a ternary operator? Do not understand